base_layer/core/src/base_node/sync/horizon_state_sync/synchronizer.rs

```diff
@@ -199,12 +199,8 @@ impl<'a, B: BlockchainBackend + 'static> HorizonStateSynchronization<'a, B> {
                         self.peer_ban_manager
                             .ban_peer_if_required(node_id, &Some(reason.clone()))
                             .await;
-
-                        if reason.ban_duration > self.config.short_ban_period {
-                            self.remove_sync_peer(node_id);
-                        }
                     }
-
+                    self.remove_sync_peer(node_id);
```

Comment:
```suggestion
                    if let BlockSyncError::MaxLatencyExceeded { .. } = err {
                        latency_counter += 1;
                    } else {
                        self.remove_sync_peer(node_id);
                    }
```
and remove the statement below